pkg/runtime/log/resource.go

```diff
@@ -163,28 +150,38 @@ func InfoResource(
 	AdaptResource(log, res, additionalValues...).V(0).Info(msg)
 }
 
-// AdaptAdoptedResource returns a logger with log values set for the adopted
-// resource's kind, namespace, name, etc
-func AdaptAdoptedResource(
-	log logr.Logger,
-	res *v1alpha1.AdoptedResource,
+// expandResourceFields returns the key/value pairs for a resource that should
+// be used as structured data in log messages about the resource
+func expandResourceFields(
+	res acktypes.AWSResource,
 	additionalValues ...interface{},
-) logr.Logger {
-	ns := res.Namespace
-	resName := res.Name
-	generation := res.Generation
-	group := res.Spec.Kubernetes.Group
-	kind := res.Spec.Kubernetes.Kind
+) []interface{} {
+	metaObj := res.MetaObject()
+	ns := metaObj.GetNamespace()
+	resName := metaObj.GetName()
+	generation := metaObj.GetGeneration()
+	rtObj := res.RuntimeObject()
+	kind := rtObj.GetObjectKind().GroupVersionKind().Kind
 	vals := []interface{}{
-		"target_group", group,
-		"target_kind", kind,
+		"kind", kind,
 		"namespace", ns,
 		"name", resName,
 		"generation", generation,
 	}
 	if len(additionalValues) > 0 {
 		vals = append(vals, additionalValues...)
 	}
+	return vals
+}
+
+// AdaptAdoptedResource returns a logger with log values set for the adopted
+// resource's kind, namespace, name, etc
+func AdaptAdoptedResource(
```

Comment:
femto nit: Maybe these logging methods can be removed now since we log directly now.
Can be done in future PRs.